### PR TITLE
Empty wallet view

### PR DESCRIPTION
**Pull request by tyler-whitman**
Closes https://github.com/helium/hotspot-app/issues/108

Simulator Screen Shot - iPhone 11 - 2021-02-02 at 16 38 33

https://user-images.githubusercontent.com/6345962/106684693-cd9e0480-657b-11eb-8823-aa4b5d642ba1.mov

### CODE REVIEW

File: src/features/wallet/root/ActivityCard/ActivityCard.tsx [hunk 1]
```diff
@@ -57,14 +56,9 @@ const ActivityCard = forwardRef((props: Props, ref: Ref<BottomSheet>) => {
   const {
     activity: { txns, filter },
   } = useSelector((state: RootState) => state)
+  const hasActivity = txns.all.data.length !== 0
   const sheet = useRef<BottomSheet>(null)
 
-  useEffect(() => {
```
Comment (tyler-whitman) [inline src/features/wallet/root/ActivityCard/ActivityCard.tsx:62]: I had to move this to WalletView in order to determine if the account had any activity or not

Comment (tyler-whitman) [reply src/features/wallet/root/ActivityCard/ActivityCard.tsx:62]: See https://github.com/helium/hotspot-app/pull/149/files#diff-356b4c10b385bf10218d349d63f2d7186b7c3ea69b220b18b4359289cf4bae25R58

File: src/features/wallet/root/WalletView.tsx [hunk 2]
```diff
@@ -46,10 +76,15 @@ const WalletView = ({ layout, animationPoints }: Props) => {
   }, [navigation])
 
   const animateActivityToBottom = useCallback(() => {
```
Comment (allenan) [inline src/features/wallet/root/WalletView.tsx:78]: maybe we can rename this `toggleShowReceive` or something since that's what it's doing in both states?

Comment (tyler-whitman) [reply src/features/wallet/root/WalletView.tsx:78]: Good point, done

**Review by allenan — APPROVED** [review]

**Review by DGKSK8LIFE — APPROVED** [review]

**Pull request merged**